<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindowPopUpForm</class>
 <widget class="QWidget" name="LoginWindowPopUpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>700</width>
     <height>400</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Log In </string>
    </attribute>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>700</width>
       <height>370</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0xDF,0xD5,0xA5);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0xDF,0xD5,0xA5);</string>
      </property>
      <widget class="QLineEdit" name="lineedit_ID">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="Login_button">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>60</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Agency FB'; font-size:20pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;DOGE  SHOP&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_error">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>210</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>290</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SIGN UP?</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>160</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Customer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Seller</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="Hidden_butt">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>160</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="btn_return">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>31</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Â«</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_error_pass">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>210</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineedit_password">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="Login_button_pass">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_identity">
       <property name="geometry">
        <rect>
         <x>204</x>
         <y>59</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="cLoginTab">
    <attribute name="title">
     <string>Customer</string>
    </attribute>
    <widget class="QLineEdit" name="c_id">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="c_password">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cAcc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="cPass">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="c_signup_button">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>270</y>
       <width>100</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="c_error_password_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>240</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="c_password2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cPass_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Retype Pasword:</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>300</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Already have an account?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>311</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Note:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Username&lt;/span&gt; accepts alphabetic chars,numbers,&lt;/p&gt;&lt;p&gt;dot'.' and dash'_'&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Password&lt;/span&gt; accepts only alphabetic chars,numbers&lt;/p&gt;&lt;p&gt;(Please implement with at least english &amp;amp; number&lt;/p&gt;&lt;p&gt;and length between 6 to 12 units)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="c_error_username_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolbtn_c_revealPass">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolbtn_c_revealPass2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>210</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sLoginTab">
    <attribute name="title">
     <string>Seller</string>
    </attribute>
    <widget class="QLineEdit" name="s_id">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="s_password">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sPass">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>96</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="sAcc">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="s_signup_button">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>270</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="s_error_password_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="s_password2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cPass_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Retype Pasword:</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>300</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Already have an account?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>311</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Note:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Username&lt;/span&gt; accepts alphabetic chars,numbers,&lt;/p&gt;&lt;p&gt;dot'.' and dash'_'&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Password&lt;/span&gt; accepts only alphabetic chars,numbers&lt;/p&gt;&lt;p&gt;(Please implement with at least english &amp;amp; number&lt;/p&gt;&lt;p&gt;and length between 6 to 12 units)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="s_error_username_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolbtn_s_revealPass2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>200</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolbtn_s_revealPass">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="s_authorizationcode">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sCode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Authorization Code:</string>
     </property>
    </widget>
    <widget class="QLabel" name="s_error_code_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>310</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="to_main_btn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>90</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
